src/components/pager/config/index.js

```diff
@@ -71,7 +72,7 @@ export const argTypes = {
   change: {
     name: "change",
     description:
-      "Dit event gebeurd wanneer een klik op een pager item gebeurd.",
+      "This event is triggered when a click on a pager item happens. In the detail of the event you can find the current page, total page number, items per page number and the total items of the pagination.",
```

Comment:
total amount of items of the pagination?

Reply:
Fixed.